fe/fe-core/src/test/java/com/starrocks/lake/StarOSAgentTest.java

```diff
@@ -731,4 +731,22 @@ public List<WorkerGroupDetailInfo> listWorkerGroup(String serviceId, List<Long>
     private Set<Long> getBackendIdsByShard(long shardId, long workerGroupId) throws UserException {
         return starosAgent.getAllNodeIdsByShard(shardId, workerGroupId, false);
     }
+
+    @Test
+    public void testListShard() throws StarClientException, DdlException {
+        ShardInfo shardInfo = ShardInfo.newBuilder().setShardId(1000L).build();
+        List<List<ShardInfo>> infos = new ArrayList<>();
+        infos.add(Lists.newArrayList(shardInfo));
+        new Expectations() {
+            {
+                client.listShard("1", Lists.newArrayList(999L), StarOSAgent.DEFAULT_WORKER_GROUP_ID, true);
+                result = infos;
+                minTimes = 0;
```

Comment:
```suggestion
                minTimes = 1;
                maxTimes = 1;
```

Expect the interface get called once and only once.

Reply:
done